invenio_rdm_records/records/mappings/os-v2/rdmrecords/records/record-v7.0.0.json

```diff
@@ -0,0 +1,1480 @@
+                "type": "integer"
+              },
+              "downloads": {
+                "type": "integer"
+              },
+              "unique_downloads": {
+                "type": "integer"
+              },
+              "data_volume": {
+                "type": "double"
+              }
+            }
+          },
+          "all_versions": {
+            "properties": {
+              "views": {
+                "type": "integer"
+              },
+              "unique_views": {
+                "type": "integer"
+              },
+              "downloads": {
+                "type": "integer"
+              },
+              "unique_downloads": {
+                "type": "integer"
+              },
+              "data_volume": {
+                "type": "double"
+              }
+            }
+          }
+        }
+      },
+      "tombstone": {
+        "properties": {
+          "removal_reason": {
+            "properties": {
+              "@v": {
+                "type": "keyword"
+              },
+              "id": {
+                "type": "keyword"
+              },
+              "title": {
+                "type": "object",
+                "dynamic": "true"
+              }
+            }
+          },
+          "note": {
+            "type": "text"
+          },
+          "removed_by": {
+            "properties": {
+              "user": {
+                "type": "keyword"
+              }
+            }
+          },
+          "removal_date": {
+            "type": "date"
+          },
+          "citation_text": {
+            "type": "text"
+          },
+          "is_visible": {
+            "type": "boolean"
+          }
+        }
+      }
+    }
+  },
+  "settings": {
+    "index.query.default_field": [
+      "id",
+      "metadata.title",
+      "metadata.contact",
+      "metadata.contributors.affiliations.name",
+      "metadata.contributors.person_or_org.name",
+      "metadata.contributors.person_or_org.family_name",
+      "metadata.contributors.person_or_org.given_name",
+      "metadata.creators.affiliations.name",
+      "metadata.creators.person_or_org.name",
+      "metadata.creators.person_or_org.family_name",
+      "metadata.creators.person_or_org.given_name",
+      "metadata.description",
+      "metadata.formats",
+      "metadata.funding.award.identifiers.identifier",
+      "metadata.funding.award.acronym.text",
+      "metadata.funding.award.number",
+      "metadata.funding.funder.name",
+      "metadata.identifiers.identifier",
+      "metadata.locations.features.place",
+      "metadata.locations.features.description",
+      "metadata.publication_date",
+      "metadata.publisher",
+      "metadata.subjects.subject",
+      "metadata.version"
+    ],
+    "analysis": {
+      "char_filter": {
+        "strip_special_chars": {
+          "type": "pattern_replace",
+          "pattern": "[\\p{Punct}\\p{S}]",
+          "replacement": ""
+        }
```

Comment:
minor: I see that we have also removed special chars, by doing so we are affecting equations, let's say we have the following input in the title "a/b = c*d - e", with this changes if I want to search for it I would need to write "ab  cd  e", which doesn't seem very intuitive for the user. I would leave this one out from RDM, unless we can match both cases, WDYT?

Reply:
Added more fields!

Reply:
was it addressed? could someone confirm?

Reply:
Sorry I missed replying
Yes, this has been addressed by adding an `original` title sub field which doesn't use any analyzers.
So we are not matching both cases.